hello-world/wrapper/typescript/README.md

```diff
@@ -0,0 +1 @@
+# TS Wrap Template
```

Comment:
Since TS wraps work a bit differently, it would be nice to get some brief explanation of the nuances here (or a link to docs).